terraform/deployment_services.tf.dev

```diff
@@ -0,0 +1,7 @@
+data "terraform_remote_state" "prod" {
+  backend = "gcs"
+  config = {
+    bucket = "discovery-etcd-io-prod-infrastructure"
```

Comment:
I prefer pattern: `<project>-infrastructure`. It is easy to remember and simpler.